### PR TITLE
CRN.627 add labs to form

### PR DESCRIPTION
**Pull request by jhonny-m**
https://asaphub.atlassian.net/jira/software/c/projects/CRN/boards/8?modal=detail&selectedIssue=CRN-627

### CODE REVIEW

File: apps/frontend/src/network/teams/TeamOutput.tsx [hunk 1]
```diff
@@ -60,6 +67,11 @@ const TeamOutput: React.FC<TeamOutputProps> = ({ teamId }) => {
     description: 'example',
     tags: [],
   };
+  const authorization = useRecoilValue(authorizationState);
```
Comment (kmaid) [inline apps/frontend/src/network/teams/TeamOutput.tsx:70]: Are we loading all labs into memory?

Comment (jhonny-m) [reply apps/frontend/src/network/teams/TeamOutput.tsx:70]: yeah

**Comment by niezgoda** [comment]
@jhonny-m please add Jira ticket ID to the title and some description would be great !

**Comment by kmaid** [comment]
Do we need the Jira ID anymore now we are linking ticket ids in branch names? @niezgoda

**Comment by niezgoda** [comment]
> Do we need the Jira ID anymore now we are linking ticket ids in branch names? @niezgoda

@kmaid I would like to have consistent naming convention :)

File: packages/react-components/src/atoms/AsyncMultiSelect.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  id,
+  enabled = true,
+  placeholder = '',
+  noOptionsMessage,
+  values = [],
+  onChange = noop,
+}) => {
+  const [inputValues, setInputValues] = useState<OptionValue[]>(values);
+  const [validationMsg, setValidationMsg] = useState('');
+  let inputRef: AsyncSelect<OptionTypeBase, true> | null;
```
Comment (marot) [inline packages/react-components/src/atoms/AsyncMultiSelect.tsx:41]: Maybe use ``useRef`` here?

File: packages/react-components/src/molecules/LabeledMultiSelect.tsx [hunk 3]
```diff
@@ -16,15 +17,40 @@ type LabeledMultiSelectProps = {
   readonly title: ReactNode;
   readonly subtitle?: React.ReactNode;
   readonly description?: ReactNode;
-} & Exclude<ComponentProps<typeof MultiSelect>, 'id'>;
-const LabeledMultiSelect: React.FC<LabeledMultiSelectProps> = ({
+};
+export type SyncLabeledMultiSelectProps = {
```
Comment (marot) [inline packages/react-components/src/molecules/LabeledMultiSelect.tsx:21]: We can add a helper method `isSync(data: Sync... | Async...): data is Sync { return data?.async === false }` and use that in the if in `forContent`. Then we can avoid the type casts (I think)

**Review by marot — APPROVED** [review]

File: packages/react-components/src/molecules/LabeledMultiSelect.tsx (hunk 3 above)
Comment (jhonny-m) [reply packages/react-components/src/molecules/LabeledMultiSelect.tsx:21]: We are going to have another PR to refactor this 👍 
cc @kmaid

**Pull request merged**